docs/index.rst

```diff
@@ -4,6 +4,7 @@
    :hidden:
 
    contributing
+   user-manual
```

Comment:
@palvarezc should it go after or before the contributing guide?

Reply:
Probs before.... Learn how it works before messing with it 😝